src/share/classes/com/sun/javatest/regtest/exec/Agent.java

```diff
@@ -593,6 +576,17 @@ public int getId() {
         return id;
     }
 
+    /**
+     * Returns the process id of the {@code AgentServer} with which this {@code Agent}
+     * communicates. Returns {@code -1} if the process id of the {@code AgentServer}
```

Comment:
As a matter of style, it is better if the first sentence is "complete", so I suggest changing ". Returns" to " or ".

```
      * Returns the process id of the {@code AgentServer} with which this {@code Agent}
      * communicates or {@code -1} if the process id of the {@code AgentServer}
      * couldn't be determined.
```

Reply:
Done. I've updated the PR to use your suggested text.